### PR TITLE
Change information according to the rules of portugal2020

### PR DESCRIPTION
**Pull request by reinaldo89**

### CODE REVIEW

File: vale-incubacao.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```
Comment (tallnato) [inline vale-incubacao.html]: Not that relevant, but should we keep all the page names in english? 🤔 

What do you guys think?

Comment (portellaa) [reply vale-incubacao.html]: totally agree

or is it a requirement?

**Review by portellaa — CHANGES_REQUESTED** [review]
Change filename to English 🙏

File: vale-incubacao.html (hunk 1 above)
Comment (ruimartinsptl) [reply vale-incubacao.html]: I've the same opinion...

**Review by ruimartinsptl — APPROVED** [review]

File: vale-incubacao.html (hunk 1 above)
Comment (reinaldo89) [reply vale-incubacao.html]: they got in touch and say we can not call it "voucher" 🤔

Comment (reinaldo89) [reply vale-incubacao.html]: I'll call it "co-financing"

File: index.html [hunk 2]
```diff
@@ -12,11 +11,10 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
         <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
         <link href="revo-slider/css/settings-custom.min.css" rel="stylesheet" type="text/css">
+        <meta name="google-site-verification" content="_M_xiowEwm0Iwxs1kq29J-aODVUaYqzF4GQmpNHW42E" />
         <meta name="google-site-verification" content="vRpyw7ZVBJLlwUluOVFSDtA0vxX-MDmGMDcAMvc5Btw" />
```
Comment (tallnato) [inline index.html:15]: will this work? Having two `google-site-verification` ?

Comment (reinaldo89) [reply index.html:15]: yes it works, there is no problem ;)

File: css/style.css [hunk 3]
```diff
@@ -85,6 +85,43 @@ body {
 	-moz-font-smoothing: antialiased;
 }
 
+.toast__container {
```
Comment (portellaa) [inline css/style.css]: why the `__`? Are those properties applied to all the pages?

File: index.html [hunk 4]
```diff
@@ -12,11 +11,10 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
         <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
         <link href="revo-slider/css/settings-custom.min.css" rel="stylesheet" type="text/css">
+        <meta name="google-site-verification" content="_M_xiowEwm0Iwxs1kq29J-aODVUaYqzF4GQmpNHW42E" />
```
Comment (portellaa) [inline index.html:14]: Why do we have two `google-site-verification` ?

**Review by portellaa — CHANGES_REQUESTED** [review]
And please apply the beautify.rei in the toast code. 😋

**Review by ruimartinsptl — APPROVED** [review]

File: css/style.css (hunk 3 above)
Comment (reinaldo89) [reply css/style.css]: I changed "__", this applies to the homepage

**Review by portellaa — APPROVED** [review]
🚀 merge that shit

**Pull request merged**